src/test_util/mod.rs

```diff
@@ -286,10 +286,36 @@ pub fn random_metrics_with_stream(
     batch: Option<BatchNotifier>,
     tags: Option<MetricTags>,
 ) -> (Vec<Event>, impl Stream<Item = EventArray>) {
```

Comment:
This function should perform the exact same for existing callers as the previously hardcoded timestamp params are being based into the new function